packages/sprotty/src/features/projection/model.ts

```diff
@@ -77,14 +81,27 @@ export function getProjections(parent: Readonly<SParentElement>): ViewProjection
 /**
  * Compute the projected bounds of the given model element, that is the absolute position in the diagram.
  */
-export function getProjectedBounds(model: Readonly<SChildElement & BoundsAware>): Bounds {
+export function getProjectedBounds(model: Readonly<SChildElement & Projectable>): Bounds | undefined {
+  if (model.projectedBounds){
```

Comment:
The formatting is a bit messed up. Can you please re-format your code according to the settings of the project?

Reply:
The formatting is now fixed.